scripts/monitor/rano_monitor/widgets/summary.py

```diff
@@ -61,10 +68,14 @@ def update_summary(self):
             status_percents["DONE"] = 0.0
 
         package_btns.display = "MANUAL_REVIEW_REQUIRED" in status_percents
+        hint_msg.display = package_btns.display
```

Comment:
So hint is shown only if there are items that require manual review. What if not? Why not to show hint if there are some items done, and a few say are extra modalities?

Reply:
That's as good point. I can remove the conditional and keep it shown all the time